apps/balancer-tools/src/app/apr/(components)/BALPrice.tsx

```diff
@@ -0,0 +1,33 @@
+import invariant from "tiny-invariant";
+
+import { getBALPriceByRound } from "../(utils)/getBALPriceByRound";
+import { Round } from "../(utils)/rounds";
+
+export default async function BALPrice({
+  roundId,
+}: {
+  roundId: string | string[];
+}) {
+  invariant(!Array.isArray(roundId), "roundId cannot be a list");
```

Comment:
this is what we had on the Remix project to early return? should we refactor the other apps to start using this?

Reply:
yup. @luizakp do you think it's worth the time going back and refactoring the other apps to do this? what do we gain by doing it?